### ISSUE TITLE
All page content must be contained by landmarks

### ISSUE THREAD

**Issue by Dawdre** [alphagov/govuk-frontend#1949]
We have been running weekly accessibility tests on all our services using govuk frontend. We use axe browser extension to test the DOM. The breadcrumbs component is consistently showing this issue...
![image](https://user-images.githubusercontent.com/11505395/92602661-5737f780-f2a6-11ea-934f-9404252df060.png)

One of our applications has recently had a 3rd party accessibility audit carried out. This was also raised as a problem. Is this an issue that is known about?

**Comment by 36degrees** [comment]
This is a duplicate of the (previously closed) #1604. I've re-opened that issue in favour of this one as it has more context.

Although ensuring that all elements are contained by landmarks is flagged as best practise, we're not aware of having elements outside of landmarks causing any issues (at least not seen in user research) and it's our understanding that it is not a WCAG failure.
